cgi/product_multilingual.pl

```diff
@@ -2152,14 +2152,17 @@ ($$$$$$)
 ;
 
 	if ($type eq 'edit') {
-		$html .= <<HTML
+		$html .= <<"HTML"
```

Comment:
Why the "HTML" instead of HTML?

Reply:
Because without quotes it is less clear if the heredoc should be interpolated or not.
```perl
my $message = <<END_MESSAGE;
...
END_MESSAGE
```
>It works and it behaves the same as if you put double-quotes around the END_MESSAGE like in the next example, but it is less clear. I'd recommend you do not use here-document without quotes around the definition of the end-string. 
~ https://perlmaven.com/here-documents

Reply:
Interesting, I have never though of that.